locations/pipelines/apply_nsi_categories.py

```diff
@@ -11,7 +11,7 @@ def process_item(self, item, spider):
         if item.get("nsi_id"):
             return item
 
-        code = item.get("brand_wikidata")
+        code = item.get("brand_wikidata", item.get("operator_wikidata"))
```

Comment:
This is not how I want to do this long term, but I want to re write this code anyway. So fine for now.